examples/knative/powershell/kn-ps-harbor-slack/README.md

```diff
@@ -0,0 +1,112 @@
+# kn-ps-harbor-slack
+Example Knative PowerShell function for sending to a Slack webhook when the Harbor webhook feature is configured to send repository events to the [kn-go-harbor-webhook function example](https://github.com/vmware-samples/vcenter-event-broker-appliance/tree/development/examples/knative/go/kn-go-harbor-webhook).
+
+# Step 1 - Build
+
+
+Create the container image locally to test your function logic.
+
+```
+export TAG=<version>
+docker build -t <docker-username>/kn-ps-harbor-slack:${TAG} .
```

Comment:
btw: it's easier if you make this a var, e.g. IMAGE= which you can later reference in your README instead of always repeating that string.

Reply:
@embano1 Sure thing, but we consequently should consider that this is probably a new issue/PR for most of our function examples... See for example: https://github.com/vmware-samples/vcenter-event-broker-appliance/tree/development/examples/knative/powershell/kn-ps-slack-vsphere-alarm

Reply:
Like this: 

```shell
export IMAGE=<image>
export TAG=<version>
export DOCKERUSER=<docker id>
docker build -t ${DOCKERUSER}/${IMAGE}:${TAG} .
```

Reply:
Done